test/dummy/config/sample_app_postgres_database.yml

```diff
@@ -0,0 +1,14 @@
+# Database settings for postgres must be in sync with
```

Comment:
It turns out that rails 6.0 does not allow us to put yaml files in the fixtures directory https://github.com/rails/rails/pull/36884. It's a PR that @gmcgibbon sent, what a coincidence!